tensorboard/plugins/custom_scalar/tf_custom_scalar_dashboard/tf-custom-scalar-card.html

```diff
@@ -0,0 +1,503 @@
+      smoothing-enabled="[[smoothingEnabled]]"
+      smoothing-weight="[[smoothingWeight]]"
+      tooltip-sorting-method="[[tooltipSortingMethod]]"
+      ignore-y-outliers="[[ignoreYOutliers]]"
+      request-manager="[[requestManager]]"
+      runs="[[runs]]"
+      tag="[[_tagFilter]]"
+      active="[[active]]"
+      data-url="[[_dataUrl]]"
+      log-scale-active="[[_logScaleActive]]"
+      process-data="[[_createProcessDataFunction()]]"
+      color-scale="[[_colorScale]]"
+      data-series="[[_dataSeriesStrings]]"
+      symbol-function="[[_createSymbolFunction()]]"
+    >
+    </tf-line-chart-data-loader>
+  </div>
+  <div id="buttons">
+    <paper-icon-button
+      selected$="[[_expanded]]"
+      icon="fullscreen"
+      on-tap="_toggleExpanded"
+    ></paper-icon-button>
+    <paper-icon-button
+      selected$="[[_logScaleActive]]"
+      icon="line-weight"
+      on-tap="_toggleLogScale"
+      title="Toggle y-axis log scale"
+    ></paper-icon-button>
+    <paper-icon-button
+      icon="settings-overscan"
+      on-tap="_resetDomain"
+      title="Fit domain to data"
+    ></paper-icon-button>
+    <span style="flex-grow: 1"></span>
+    <template is="dom-if" if="[[showDownloadLinks]]">
+      <div class="download-links">
+        <paper-dropdown-menu
+          no-label-float="true"
+          label="series to download"
+          selected-item-label="{{_dataSeriesNameToDownload}}"
+        >
+          <paper-menu class="dropdown-content" slot="dropdown-content">
+            <template is="dom-repeat" items="[[_dataSeriesStrings]]" as="dataSeriesName">
+              <paper-item no-label-float=true>[[dataSeriesName]]</paper-item>
+            </template>
+          </paper-menu>
+        </paper-dropdown-menu>
+        <a
+          download="[[_dataSeriesNameToDownload]].csv"
+          href="[[_csvUrl(_nameToDataSeries, _dataSeriesNameToDownload)]]"
+        >CSV</a> <a
+          download="[[_dataSeriesNameToDownload]].json"
+          href="[[_jsonUrl(_nameToDataSeries, _dataSeriesNameToDownload)]]"
+        >JSON</a>
+      </div>
+    </template>
+  </div>
+  <div id="matches-container">
+    <div id="matches-list-title">
+      <template is="dom-if" if="[[_dataSeriesStrings.length]]">
+        <paper-icon-button
+          icon="[[_getToggleMatchesIcon(_matchesListOpened)]]"
+          on-click="_toggleMatchesOpen"
+          class="toggle-matches-button">
+        </paper-icon-button>
+      </template>
+
+      <span class="matches-text">
+        Matches ([[_dataSeriesStrings.length]])
+      </span>
+    </div>
+    <template is="dom-if" if="[[_dataSeriesStrings.length]]">
+      <iron-collapse opened="[[_matchesListOpened]]">
+        <div id="matches-list">
+          <template
+              is="dom-repeat"
+              items="[[_dataSeriesStrings]]"
+              as="seriesName"
+            >
+              <div
+                  class="match-list-entry"
```

Comment:
go/htmlstyle should, generally speaking, take precedence when creating new files. There's no stylistic disadvantage to putting this on the previous line and then aligning the attrs.

Reply:
Done.